spec/index.bs

```diff
@@ -6659,19 +6705,17 @@ dictionary GPUQuerySetDescriptor : GPUObjectDescriptorBase {
 };
 </script>
 
-<dl dfn-type=dict-member dfn-for=GPUQuerySetDescriptor>
-    : <dfn>pipelineStatistics</dfn>
-    ::
-        The set of {{GPUPipelineStatisticName}} values in this sequence defines which pipeline statistics will be returned in the new query set.
+<div class=validusage dfn-for=GPUQuerySetDescriptor>
+    <dfn abstract-op>GPUQuerySetDescriptor Valid Usage</dfn>
 
-        <div class=validusage dfn-for=GPUQuerySetDescriptor.pipelineStatistics>
-            <dfn abstract-op>Valid Usage</dfn>
+    Given a {{GPUQuerySetDescriptor}} |this| the following validation rules apply:
 
-            1. |pipelineStatistics| is ignored if type is not {{GPUQueryType/pipeline-statistics}}.
-            2. If {{GPUFeatureName/pipeline-statistics-query}} is not available, |type| must not be {{GPUQueryType/pipeline-statistics}}.
-            3. If |type| is {{GPUQueryType/pipeline-statistics}},  |pipelineStatistics| must be a sequence of {{GPUPipelineStatisticName}} values which cannot be duplicated.
-        </div>
-</dl>
+    1. |this|.{{GPUQuerySetDescriptor/type}} must be a [=valid=] {{GPUQueryType}}.
```

Comment:
nit: Please use (`-`) instead of numbers (`1.`) for validation rules since they're conceptually unordered.